gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -1110,6 +1116,30 @@ message TripModifications {
   extensions 9000 to 9999;
 }
 
+message ReplacementVehicle {
+  // The route type of the replacement vehicle. If not specified, the route type of the original trip will be used.
+  optional int32 route_type = 1;
+
+  // If the replacement vehicle is not following the schedule from the original trip, a new headway can be specified.
+  // The service is assumed to run from the arrival_time of the first modified trip at the start_stop_sequence to the arrival_time of the last modified trip at the start_stop_sequence.
+  optional int32 headway_secs = 2;
+
+  // Transfer time from and to the first and last stop of the modification, if required.
+  // TODO : ability to specify the full transfers.txt capabilities 
+  optional int32 to_first_replacement_stop_transfer_time = 3;
+  optional int32 from_last_replacement_stop_transfer_time = 4;
+
+  // TODO add definition on how trip_id is modified for replacement vehicle. 
```

Comment:
A few thoughts on this:

1. We'll need a way to have a one to many relationship between a replaced trip and the replacement trip(s). I believe the Trip Modifications proposal allows just one to one.
2. I could see cases where a producer would want to explicitly list out the replacement trip_id's (for example, a commuter rail trip replaced by 3 different bus trips) and other cases where listing all replacement trip_id's would be tough (for example, a headway-based replacement service).